builtin/common/tests/vector_spec.lua

```diff
@@ -4,14 +4,14 @@ dofile("builtin/common/vector.lua")
 describe("vector", function()
 	describe("new()", function()
 		it("constructs", function()
-			assert.same({ x = 0, y = 0, z = 0 }, vector.new())
-			assert.same({ x = 1, y = 2, z = 3 }, vector.new(1, 2, 3))
-			assert.same({ x = 3, y = 2, z = 1 }, vector.new({ x = 3, y = 2, z = 1 }))
+			assert.same({ 0, 0, 0 }, vector.new())
+			assert.same({ 1, 2, 3 }, vector.new(1, 2, 3))
+			assert.same({ 3, 2, 1 }, vector.new({ x = 3, y = 2, z = 1 }))
```

Comment:
should have a test for the metatable

Reply:
I thought that, too. But in http://www.lua.org/manual/5.1/manual.html#2.8 section "eq", it says that it already compares metatables in `getcomphandler` each time `equal` is used.

Reply:
No I meant like: `assert.is_true(getmetatable(vector.new()) ~= nil)`

Reply:
Done using the new vector.is_vector.